app/models/track.rb

```diff
@@ -13,6 +13,9 @@ class Track < ApplicationRecord
   has_many :concept_exercises # rubocop:disable Rails/HasManyOrHasOneDependent
   has_many :practice_exercises # rubocop:disable Rails/HasManyOrHasOneDependent
 
+  # TODO: Pre-launch: remove dependent: :destroy
+  has_many :tasks, class_name: "Github::Task", dependent: :destroy
```

Comment:
The dependent bit was required according to Rails. I tried it without and got an error.

Reply:
Rails or Rubocop?

Reply:
I don't remember :D